<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefPanel</class>
 <widget class="QWidget" name="PrefPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrefPanel</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QCheckBox" name="cBoxFullscreen">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>35</y>
          <width>212</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Start Joker in fullscreen mode</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="cBoxLastFile">
        <property name="geometry">
         <rect>
          <x>53</x>
          <y>12</y>
          <width>179</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Remember last open file</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabStrip">
       <attribute name="title">
        <string>Strip</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="layoutStrip">
          <item row="2" column="0">
           <layout class="QVBoxLayout" name="layoutFont">
            <item>
             <widget class="QLineEdit" name="lineEditFilter">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowserFont"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QFormLayout" name="layoutBoldness">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::FieldsStayAtSizeHint</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblBoldness">
              <property name="text">
               <string>Bold :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="sliderBoldness">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1" alignment="Qt::AlignHCenter">
           <widget class="QSlider" name="sliderStripHeight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblStripHeight">
            <property name="text">
             <string>Strip Height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QFormLayout" name="layoutItalic">
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="lblItalic">
              <property name="text">
               <string>Italic :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="sliderItalic">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QFormLayout" name="layoutForm">
            <item row="0" column="0">
             <widget class="QLabel" name="lblSpeed">
              <property name="text">
               <string>Speed:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxSpeed">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>12</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
      <widget class="QWidget" name="tabSynchro">
       <attribute name="title">
        <string>Synchronisation</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="layoutSynchro">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblDelay">
            <property name="text">
             <string>Delay (ms)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxDelay">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblQuarterFram">
            <property name="text">
             <string>UseQuarterFram</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cBoxQuarterFram">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblSonyAutoconnect">
            <property name="text">
             <string>Sony autoconnect</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="cBoxSonyAutoconnect">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblSonyID">
            <property name="text">
             <string>Sony ID</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditSonyID">
            <property name="placeholderText">
             <string>F0 C0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblSonyHighSpeed">
            <property name="text">
             <string>Sony high speed</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBoxSonyHighSpeed">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
